attestation-service/Makefile

```diff
@@ -29,15 +28,24 @@ endif
 build: grpc-as restful-as
 
 grpc-as:
-	cargo build --bin grpc-as $(release) $(default-features) --features grpc-bin$(OPTIONAL_FEATURES)
+	cargo build --bin grpc-as $(release) $(default-features) --features grpc-bin,$(VERIFIER)
 
 restful-as:
-	cargo build --bin restful-as $(release) $(default-features) --features restful-bin$(OPTIONAL_FEATURES)
+	cargo build --bin restful-as $(release) $(default-features) --features restful-bin,$(VERIFIER)
```

Comment:
You removed the definition of `default-features`. If `FEATURES` were specified, then `--no-default-features` was enabled as well. [`all-verifier` is a part of the default features](https://github.com/confidential-containers/trustee/blob/fafa3412bb34e93a7cbf50d51c4dca824fa771a4/attestation-service/Cargo.toml#L7), so without `--no-default-features` enabled, all of the verifiers will be enabled regardless of what is specified in `VERIFIERS`.